wpeverest/everest-forms
# Enhancement - Custom file name settings

### All Submissions:






### Changes proposed in this Pull Request:



Previously, file name are generated randomly. This PR gives user to add custom file name.

### How to test the changes in this Pull Request:

1. Creare new form and drag file upload.
2. Go to File upload >> Field Options >> Custom File name
3. Insert custom name.
4. Preview form and and upload file.
5. Check entries whether custom file name is set or not.

Dependency [PR](https://github.com/wpeverest/everest-forms-pro/pull/916)
### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Custom file name settings
